driver-core/src/main/resources/META-INF/native-image/reflect-config.json

```diff
@@ -0,0 +1,96 @@
+[
+{
+  "name":"com.mongodb.BasicDBObject",
```

Comment:
We have a slightly different set of reflection registrations. But, I believe yours are more accurate.